addons/mail/views/mail_channel_partner_views.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<odoo>
+    <data>
+        <record model="ir.ui.view" id="mail_channel_partner_short_view_account">
```

Comment:
Trump said "America first". I say "identifier first, then model" ;)
And the id should be like `res_partner_view_form_account`